### PR TITLE
Improving storybook docs

### PR DESCRIPTION
**Pull request by marcinsawicki**
## Description
Improving storybook docs for following components:
- ActionMenu
- Avatar
- Badge
- Button
- Card
- Checkbox
- DatePicker
- Icon
- Input
- Loader
- Modal

## Storybook

https://feature-storybook-docs-pt1--613a8e945a5665003a05113b.chromatic.com

## Checklist

**Obligatory:**

- [x] Self review (use this as your final check for proposed changes before requesting the review)
- [x] Add reviewers (`livechat/design-system`)
- [x] Add correct label
- [ ] Assign pull request with the correct issue

### CODE REVIEW

File: packages/react-components/src/components/DatePicker/DatePicker.mdx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+```jsx
+<DatePicker
+  fromMonth={new Date('06/20/2021')}
+  toMonth={new Date('09/20/2021')}
+  month={new Date('09/20/2021')}
+/>
+```
+
+## Component API <a id="ComponentAPI" />
+<ArgsTable />
```
Comment (VadymBezpalko) [inline packages/react-components/src/components/DatePicker/DatePicker.mdx:24]: The Component API here shows only two params: `innerRef` and `range`. I think you can use here a `<ArgTypes of={DatePickerStories.DatePicker} />` to show props related to this basic story

File: packages/react-components/src/components/DatePicker/DatePicker.mdx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import { Canvas, Story, ArgsTable, Meta, Title } from '@storybook/blocks';
+import { Figma } from "@storybook/addon-designs/blocks";
+import * as DatePickerStories from './DatePicker.stories';
+import { BlockColumnsContainer, BlockColumns } from '../../stories/components/BlockColumns';
```
Comment (VadymBezpalko) [inline packages/react-components/src/components/DatePicker/DatePicker.mdx:4]: This import is not used

File: packages/react-components/src/components/DatePicker/DatePicker.mdx [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import { Canvas, Story, ArgsTable, Meta, Title } from '@storybook/blocks';
```
Comment (VadymBezpalko) [inline packages/react-components/src/components/DatePicker/DatePicker.mdx:1]: `Story` component is not used

**Review by VadymBezpalko — APPROVED** [review]
👍

**Comment by vladko-uxds** [comment]
✨

**Review by vladko-uxds — APPROVED** [review]

**Pull request merged**